tests/NLog.UnitTests/Config/ExtensionTests.cs

```diff
@@ -281,6 +281,30 @@ public void ExtensionTest_extensions_not_top_and_used()
         }
 
         [Fact]
+        public void ExtensionShouldThrowNLogConfiguratonExceptionWhenRegisteringInvalidType()
+        {
+            var configXml = @"
```

Comment:
You need to set InternalLogger.throwsconfigexceptions on true

(without typo, on mobile now ;)